src/Translatable/Translatable.php

```diff
@@ -458,4 +459,35 @@ public function __isset($key)
     {
         return $this->isTranslationAttribute($key) || parent::__isset($key);
     }
+
+    public function getTranslationChanges(): array
+    {
+        switch (config('translatable.rule_factory.format')) {
+            case RuleFactory::FORMAT_KEY:
+                $replacementFunc = function (string $key, string $locale): string {
+                    return $key . ':' . $locale;
+                };
+                break;
+            default:
+                $replacementFunc = function (string $key, string $locale): string {
+                    return $locale . '.' . $key;
+                };
+        }
```

Comment:
Could we move that block in a static method on the `RuleFactory` class.
This could also replace the current `RuleFactory->getReplacement()` method.
https://github.com/Astrotomic/laravel-translatable/blob/e3aac483509ffbf425915a4bedec58f7095c58de/src/Translatable/Validation/RuleFactory.php#L126-L135

Reply:
Good point, I also missed the DRYness of my solution, but didn't want to be too intrusive with my PR ;-)

Reply:
As this is no more need for this PR I would suggest to move it to a separate PR (if still considered useful).